_studio/shared/src/libmfx_core_vaapi.cpp

```diff
@@ -893,7 +896,7 @@ VAAPIVideoCORE::DoFastCopyWrapper(
     {
         if (srcMemType & MFX_MEMTYPE_SYSTEM_MEMORY)
         {
-            if (LumaIsNull(pSrc))
```

Comment:
Have you deleted LumaIsNull macro/function?

Reply:
No, it is still used elsewhere.